akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ProxyShardingSpec.scala

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster.sharding
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import akka.actor.ActorRef
+import akka.testkit.AkkaSpec
+import akka.actor.ActorSystem
+import akka.testkit.TestActors
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+import scala.concurrent.duration.FiniteDuration
+
+class ProxyShardingSpec extends AkkaSpec {
+
+  val role = "Shard"
+  val nodes = 5
+
+  val configString =
+    "akka.actor.provider=\"akka.cluster.ClusterActorRefProvider\""
+  val config: Config = ConfigFactory
+    .parseString(configString)
+    .withFallback(ConfigFactory.defaultApplication)
+  val mySystem = ActorSystem("mySystem", config)
```

Comment:
ActorSystem is already started by `AkkaSpec` and can accessed with `system`. What is the reason for starting another system?